MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -339,11 +339,13 @@ static inline bool mvkVkComponentMappingsMatch(VkComponentMapping cm1, VkCompone
 #define mvkPrintSizeOf(type)    printf("Size of " #type " is %lu.\n", sizeof(type))
 
 
-#pragma mark -
-#pragma mark Template functions
-
 #pragma mark Math
 
+/** Rounds the value to nearest integer using half-to-even rounding. */
+static inline double mvkRoundHalfToEven(const double val) {
+	return val - std::remainder(val, 1.0);	// remainder() uses half-to-even rounding
```

Comment:
I guess, unlike in MSL, `rint(3)` isn't guaranteed to perform nearest-even rounding, even when `fegetround(3)` returns `FE_TONEAREST`.